ads/google/a4a/experiment-utils.js

```diff
@@ -63,12 +63,13 @@ export class ExperimentUtils {
    * @return {?string}
    */
   maybeSelectExperiment(win, element, selectionBranches, experimentName) {
-    const experimentInfoMap = /** @type {!Object<string, !ExperimentInfo>} */ ({});
-    experimentInfoMap[experimentName] = {
+    const experimentInfoList = /** @type {!Array<!ExperimentInfo>} */ ([]);
```

Comment:
do we still need to cast?

Reply:
We need to cast here even more, since the compiler won't know what the element type of `[]` is.

Reply:
Ok, I was hoping it was smart enough to figure it out when we call `randomlySelectUnsetExperiments`